### ISSUE TITLE
Question: rejoiner inside existing grpc service? 

### ISSUE THREAD

**Issue by sshevlyagin** [google/rejoiner#130]
I can't tell how possible this is from the examples. If I have an existing GRPC service that's using io.grpc and guice, could I use rejoiner to expose a graphql endpoint from the existing service and leverage some of the auto conversion?

**Comment by siderakis** [comment]
Yeah, you can create the graphed schema and serve a graphql endpoint in your existing service.

**Comment by sshevlyagin** [comment]
Awesome, i'll let you know if I do anything here.
